resources/migrations/001_update_migrations.yaml

```diff
@@ -5823,6 +5823,62 @@ databaseChangeLog:
         - sql:
             sql: >-
               DROP VIEW IF EXISTS v_content;
+  - changeSet:
+      id: v50.2024-02-29T15:06:43
+      author: tsmacdonald
+      comment: Add the field_usage join table
+      changes:
+        - createTable:
+            tableName: field_usage
+            remarks: Fields used by a card
+            columns:
+              - column:
+                  name: id
+                  remarks: PK
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: card_id
+                  remarks: referenced card
+                  type: int
+                  constraints:
+                    nullable: false
+                    referencedTableName: report_card
+                    referencedColumnNames: id
+                    foreignKeyName: fk_field_usage_card_id
+                    deleteCascade: true
+              - column:
+                  name: field_id
+                  remarks: referenced field
+                  type: int
+                  constraints:
+                    nullable: false
+                    referencedTableName: metabase_field
+                    referencedColumnNames: id
+                    foreignKeyName: fk_field_usage_field_id
+                    deleteCascade: true
+              - column:
+                  remarks: Name of the used column, as it is in the query (i.e., the old name)
```

Comment:
> When the name changes though, don't we create a new column anyway though? I don't understand how this could be "stale".

You're right, but I'm not filled with confidence about depending on that forever. If you think I'm being a coward I'm open to reconsidering though